src/crypto/tls/common.go

```diff
@@ -440,10 +468,18 @@ type CertificateRequestInfo struct {
 	// empty slice indicates that the server has no preference.
 	AcceptableCAs [][]byte
 
+	// SupportDelegatedCredential is true if the server indicated willingness
+	// to negotiate the delegated credential extension.
+	SupportDelegatedCredential bool
```

Comment:
Same as above.

Reply:
Mmm.. might be. I see that in other places of the lib, they use 'Supports'

Reply:
Yeah, it's sort of a shorthand for "it supports delegated credentials" whereas in the config struct it'd be "please support delegated credentials".